packages/remark-lint-list-item-indent/index.js

```diff
@@ -167,14 +167,16 @@ const remarkLintListItemIndent = lintRule(
     url: 'https://github.com/remarkjs/remark-lint/tree/main/packages/remark-lint-list-item-indent#readme'
   },
   /** @type {import('unified-lint-rule').Rule<Root, Options>} */
-  (tree, file, option = 'tab-size') => {
+  (tree, file, option = 'tab') => {
     const value = String(file)
+    option =
+      {'tab-size': 'tab', space: 'one'}[/** @type {never} */ (option)] || option
```

Comment:
No, should be backwards compatible. Maybe support for all but the preferred literals could be dropped at some future point ... Or not.

Reply:
Hmm, I guess I missed this line?

Your object is nice but it’s a) a bit hard to read, and b) it’s vulnerable to things like `toString` or `constructor` being passed (which here doesn’t seem to result in actual problems, but it’s often good to steer clear of it anyway):
How about:

```
// To do in major: remove legacy fallbacks.
if (option === 'tab-size') {
  option = 'tab'
} else if (option === 'space') {
  option === 'one'
}
```

Reply:
- [x] I didn't think of that (other properties of "empty" objects), thanks! I've replaced it with:
  ```js
    option =
      option === 'tab-size' ? 'tab'
        : option === 'space' ? 'one'
        : option